requirements.txt

```diff
@@ -1,2 +1,3 @@
 pytest
 six
+docker-py
```

Comment:
I would keep docker-py as an optional requirements, like paramiko and salt are.

See testinfra/backend/paramiko.py for a try/except import example